src/main/java/org/folio/rest/impl/InvoicesImpl.java

```diff
@@ -174,6 +188,40 @@ public void deleteInvoiceInvoiceLinesById(String id, String lang, Map<String, St
       .exceptionally(t -> handleErrorResponse(asyncResultHandler, invoiceLineHelper, t));
   }
 
+  @Override
+  public void putInvoiceInvoiceLinesFundDistributionsValidate(ValidateFundDistributionsRequest entity,
+     Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+      Context vertxContext) {
+    try {
+      MonetaryAmount subTotal;
+      Double total = null;
+      CurrencyUnit currencyUnit = Monetary.getCurrency(entity.getCurrency());
+      List<FundDistribution> fundDistributionList = entity.getFundDistribution();
+      if(CollectionUtils.isNotEmpty(entity.getAdjustments()))
+      {
+      if (InvoiceValidator.isAdjustmentIdsNotUnique(entity.getAdjustments())) {
+       throw new HttpException(400,ADJUSTMENT_IDS_NOT_UNIQUE);
+      }
+      subTotal = Money.of(entity.getSubTotal(), currencyUnit);
+      MonetaryAmount adjustmentAndFundTotals = HelperUtils.calculateAdjustmentsTotal(entity.getAdjustments(), subTotal);
+      total = HelperUtils.convertToDoubleWithRounding(adjustmentAndFundTotals.add(subTotal));
```

Comment:
You can declare total variable here (in place of usage)

Reply:
i guess we can not, aswe are using "total" variable outside of the if else scope